src/main/java/tasks/TaskList.java

```diff
@@ -0,0 +1,88 @@
+package tasks;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+
+    private List<Task> list;
+
+    public TaskList(List<Task> loadedData) {
+        this.list = loadedData;
+    }
+
+    public TaskList() {
+        this.list = new ArrayList<>();
+    }
+
+    /**
+     * Adds the specified task to the task list.
+     *
+     * @param task Task to be added.
+     * @param saved True if the task to added should be pre-marked as done, otherwise False.
+     */
+    public void add(Task task, boolean saved) {
```

Comment:
```suggestion
    public void addTask(Task task, boolean saved) {
```

Same as the above, can be more descriptive with your method names.